feature/amazon/src/main/java/com/revenuecat/purchases/amazon/AmazonBilling.kt

```diff
@@ -73,6 +73,11 @@ internal class AmazonBilling constructor(
         connected = true
     }
 
+    override fun startConnectionOnMainThread(delayMilliseconds: Long) {
+        // no special threading needed for Amazon IAP
+        startConnection()
```

Comment:
I think even though special threading is not needed this function should still be calling `startConnection` in the main thread. Someone can call this function in the future thinking it would call start connection in the main thread without checking its implementation, and the function might not be doing what they want.

Reply:
yeah fair point, will do!